src/Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorApp.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import { ERROR_MESSAGES } from "Components/ErrorPage"
+import { ArtworkErrorApp_artworkError$data } from "__generated__/ArtworkErrorApp_artworkError.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import { OtherWorksQueryRenderer } from "Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorAppOtherWorks"
+import { RelatedWorksQueryRenderer } from "Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorAppRelatedWorks"
+
+interface ArtworkErrorAppProps {
+  artworkError: ArtworkErrorApp_artworkError$data
+}
+const ArtworkErrorApp = (props: ArtworkErrorAppProps) => {
```

Comment:
```
const ArtworkErrorApp: ReactFC<ArtworkErrorAppProps>
```

Also, can we name this `Artwork404` (and no need for "App" in name, unless yr going to move it into src/Apps/ArtworkErrorApp`

Reply:
I chose the `App` suffix b/c the top-level basically branches between `ArtworkApp` and this component based on if there's an error or not, so in the heirarchy it's kind of a sibling to `ArtworkApp`.

I'll take another pass at the naming/organization today. I'm not the best at that w/ FE components so will do a second draft.